index.html

```diff
@@ -290,6 +111,51 @@ <h2 class="title1"> Contact me </h2>
         </div>
       </section>
   <!-- End Contact Form  -->
+  <!-- Popup section -->
+        <div class="popuphidden hidden">
+            <div class="popupcard">
+                <div class="pophead">
+                  <img class="popclose" id="closeModal" src="./img/close.png">
+                    <h2 class="poptitle"> Tonic </h2>
+                    <div class="popdetail">
+                        <p class="company">CANOPY</p>
+                        <img src="./img/Counter.png" alt="">
+                        <p class="jobdescription">Back End Dev</p>
+                        <img src="./img/Counter.png" alt="">
+                        <p class="year">2015</p>
+                    </div>
+                </div>
+                <div class="popimg">
+                    <img class="popimge-mob" src="./img/Snapshoot-Portfolio.png">
+                    <img class="popimge-desk" src="./img/SnapshootPortfoliopopup.png">
+                </div>
+                <div class="sect">
+                <div class="popdescription">
+                    <p class="popdescrip"> Bla bla bla</p>
+                </div>
+                <div class="popbuttons">
+                    <div class="poptec">
+                        <ul class="popsmallbuttons">
+                            <li><button class="pop-button-small">html</button></li>
+                            <li><button class="pop-button-small">css</button></li>
+                            <li><button class="pop-button-small">JavaScript</button></li>
+                        </ul>
+                    </div>
+                    <div class="modal-buttons">
+                        <a href="#" class="btn img-btn live">
+                                   See live
+                        <img src="./img/icon.png" alt="See live">
+                        </a>
+                          <a href="#" class="btn img-btn source">
+                            See Source
+                         <img src="./img/vector-github.png" alt="See source" class="source-image">
+                         </a>
+                    </div>
+                </div>
+            </div>
+            </div>
+        </div>
```

Comment:
- [ ] Kindly notice that you should follow the Figma design to make your portfolio professional and responsive, so please make the project popup window and the projects like the Figma desing.

Your Design | Figma Design
------------- | ---------------
![Capture7](https://user-images.githubusercontent.com/33752735/189239500-e549437b-4a34-40c3-8324-2b77d1b704d9.PNG) | ![Capture8](https://user-images.githubusercontent.com/33752735/189239524-6137eeb4-a0f0-4783-8a00-0accaf74ae81.PNG)

Your Desing | Figma Design
------------- | ----------------
![Capture10](https://user-images.githubusercontent.com/33752735/189239594-95a4e615-569a-48c9-80c0-7a0b3fc43c0e.PNG) | ![Capture9](https://user-images.githubusercontent.com/33752735/189239615-3f957a64-ad11-445e-b509-f7a7d68f6c9e.PNG)